src/composables/useFlawAffectsModel.ts

```diff
@@ -96,18 +96,17 @@ export function useFlawAffectsModel(flaw: Ref<ZodFlawType>) {
     affectsToDelete.value = [];
   }
 
-  function addBlankAffect() {
+  function addBlankAffect(moduleName: string, callback: (affect: ZodAffectType) => void) {
```

Comment:
```suggestion
  function addBlankAffect(): ZodAffectType {
    const embargoed = flaw.value.embargoed;
    flaw.value.affects.push({
      embargoed,
      affectedness: '',
      resolution: '',
      delegated_resolution: '', // should this be null
      ps_module: '',
      ps_component: '',
      impact: '',
      cvss_scores: [{
        // affect: z.string().uuid(),
        cvss_version: 'V3',
        issuer: 'RH',
        comment: '',
        score: null,
        vector: '',
        embargoed,
        alerts: [],
      }],
      trackers: [{ errata: [] }],
      alerts: [],
    } as ZodAffectType);
    return flaw.value.affects.at(-1);
  }
```

Reply:
This would avoid having to add a callback by just returning the new affect that was just added to the flaw's list of affects, this should be reactive